lms/static/js/learner_dashboard/views/program_header_view.js

```diff
@@ -40,10 +40,18 @@ class ProgramHeaderView extends Backbone.View {
         return logo;
     }
 
+    getIsSubscribed() {
+        return (
+            this.model.get('isSubscriptionEligible') &&
+            this.model.get('subscriptionData')?.[0]?.subscription_state === 'active'
```

Comment:
nit: another place where I find the line a bit hard to read due to how many things are happening. I'd love to see this split up and a bit more readable

Reply:
Sure